src/index.d.ts

```diff
@@ -982,6 +982,121 @@ declare namespace admin.remoteConfig {
    */
   type RemoteConfigParameterValue = ExplicitParameterValue | InAppDefaultValue;
 
+  /**
+   * Interface representing a Remote Config template version.
+   * Contains all metadata about a particular version of the Remote Config template.
+   * All fields are set at the time the specified Remote Config template was written.
+   */
+  export interface Version {
+    /**
+     * The version number of the version's corresponding Remote Config template. Output only.
+     */
+    versionNumber?: string | number;
+    
+    /**
+     * The Remote Config template creation timestamp in RFC3339 UTC "Zulu" format,
+     * accurate to nanoseconds. Output only.
+     */
+    updateTime?: string;
+
+    /**
+     * The source of origin of the template update action. Output only.
+     */
+    updateOrigin?: ('REMOTE_CONFIG_UPDATE_ORIGIN_UNSPECIFIED' | 'CONSOLE' |
+      'REST_API' | 'ADMIN_SDK_NODE');
+
+    /**
+     * The type of the template update action. Output only.
+     */
+    updateType?: ('REMOTE_CONFIG_UPDATE_TYPE_UNSPECIFIED' |
+      'INCREMENTAL_UPDATE' | 'FORCED_UPDATE' | 'ROLLBACK');
+
+    /**
+     * Aggregation of all metadata fields about the account that performed the update. Output only.
```

Comment:
`Metadata about the user that published this version.`

Just curious. What does this get set to for versions published by the Admin SDK?

Reply:
It is set to something similar to:
```
{
  email:
   'firebase-adminsdk-vs7zk@PROJECT_ID.iam.gserviceaccount.com',
  imageUrl:
   'https://coded_url/photo.jpg'
}
```